spec/jobs/google_scraping_job_management_job_spec.rb

```diff
@@ -0,0 +1,38 @@
+# frozen_string_literal: true
+
+require 'rails_helper'
+
+RSpec.describe GoogleScrapingJobManagementJob, type: :job do
+  include ActiveJob::TestHelper
+  ActiveJob::Base.queue_adapter = :test
+
+  describe '#perform' do
+    context 'given valid inserted keywords' do
+      it 'enqueues google scraping job manangement job' do
```

Comment:
this sentence is quite hard to understand the phrase `google scraping job manangement job`. Should we think of an alternative?

Reply:
Yes I think so, one of the most difficult part in programming is naming haha.
Do you have any suggestion?

Reply:
How do you think about `InitiateGoogleScrapingJob`?

Reply:
Fixed in 366deb3f2f356c80eb446aa753e9d2f4dbea10e4

Reply:
Fixed in 60b0ae5997ee6139477163160f463694ead047c1